packages/manager/modules/exchange/src/exchange.tabs.controller.js

```diff
@@ -90,25 +93,28 @@ export default class ExchangeTabsCtrl {
           {
             type: 'SEPARATOR',
           },
-          {
-            label: $translate.instant('exchange_action_configuration'),
-            type: 'ACTION',
-            fn: () => {
-              navigation.setAction('exchange/configure/service-configure');
-            },
-          },
         );
       }
 
       if (
-        $routerParams.tab &&
-        this.tabs.indexOf($routerParams.tab.toUpperCase())
+        this.$routerParams.tab &&
+        this.tabs.indexOf(this.$routerParams.tab.toUpperCase())
       ) {
-        this.selectedTab = $routerParams.tab.toUpperCase();
+        this.currentTab = this.$routerParams.tab.toUpperCase();
       }
 
-      this.updateScope();
-      this.changeTab(this.selectedTab);
+      this.changeTab(this.currentTab);
+    });
+
+    this.services.$scope.$on('$locationChangeSuccess', () => {
```

Comment:
Why did you change all this part ?

Reply:
To allow to update tabs when url is updated.